tests/components/repairs/test_init.py

```diff
@@ -463,6 +463,7 @@ async def test_delete_issue(
     }
 
 
+@pytest.mark.no_fail_on_log_exception
```

Comment:
These are expected to generate logged exceptions because the platform intentionally raises. Now that we use the helper, the test framework knows about the exception so we need to mark it to not fail.